qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -39,43 +42,42 @@ public void verifyModulesAvailableOnHomePage() {
         assertTrue(homePage.isSystemAdministrationAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on homepage after login as clerk")
     public void verifyClerkModulesAvailableOnHomePage() {
         goToLoginPage().loginAsClerk();
-        initiateHomePage();
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
         assertTrue(homePage.isRegisterPatientCustomizedForRefAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as doctor")
     public void verifyDoctorModulesAvailableOnHomePage() {
         goToLoginPage().loginAsDoctor();
-        initiateHomePage();
         assertTrue(homePage.isFindAPatientAppPresent());
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as nurse")
     public void verifyNurseModulesAvailableOnHomePage() {
         goToLoginPage().loginAsNurse();
-        initiateHomePage();
         assertTrue(homePage.isFindAPatientAppPresent());
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
         assertTrue(homePage.isCaptureVitalsAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as sysadmin")
```

Comment:
How is login as admin different from login as sysadmin?

Reply:
i think the confusion is with in login page where this was made. admin like you had said is the super user. then for this case we have sysadmin and SysAdmin123. i think these are two different cases from where they differ.

Reply:
Anything that has `admin` is super user.  Else, you are creating unnecessary confusion.

Reply:
I get your point and where the confusion is. The thing is this is right but the naming of gherkin is bringing confusion, the difference is sysadmin can only access to the system administration app and appointment scheduling but admin can access all the apps.Does this make sense.

Reply:
can we however change the naming of gherkin

Reply:
@dkayiwa does it now add a difference.

Reply:
It will never make sense as long as an admin and sysadmin are different.

Reply:
@dkayiwa  How about  this [commit](https://github.com/openmrs/openmrs-contrib-qaframework/pull/369/commits/ff309d05402b6f9d3e9bee298363b63785259817) where by i added login as admin to differentiate it from loginAsSysadmin.

Reply:
admin, sysadmin, and super user are all the same. Else, you are creating confusing tests.

Reply:
Ohh so are you suggesting we remove the one for sysadmin.

Reply:
i removed the one for sysadmin. because it kind of duplicate.

Reply:
@dkayiwa  hope this looks fair.